ash1eygrace/giphy-search
# Add new Spinner.js component and use it while waiting for search results in App.js

This PR adds a new Spinner.js component that displays a GIF in the middle of the screen to animate the loading screen on slower devices vs. displaying text in the left-hand corner of the page body that says "loading..." 

## Now: 
 
When the gifs are loading, you see the <p>loading..</p> text in App.js: 

https://user-images.githubusercontent.com/29527450/221591182-d3d6e9fd-38a8-4b33-a527-dd844de72bd9.mov

## After: 

With the new Spinner.js component, you'll see this: 

https://user-images.githubusercontent.com/29527450/221592638-ff45ca3b-7466-4916-9888-47256892a6b7.mov

## Testing Instructions 

1. Open the app in your browser 
2. Open Developer tools > network > and throttle the network to Slow or Fast 3G 
3. The spinner should load instead of the paragraph.
